typewise_alert.test.py

```diff
@@ -3,9 +3,16 @@
 
 
 class TypewiseTest(unittest.TestCase):
-  def test_infers_breach_as_per_limits(self):
-    self.assertTrue(typewise_alert.infer_breach(20, 50, 100) == 'TOO_LOW')
+    def test_check_and_alert_passive_cooling(self):
+        self.assertTrue(typewise_alert.check_and_alert('TO_CONTROLLER', 'PASSIVE_COOLING', 40)[0] == 'TOO_HIGH')
+        self.assertTrue(typewise_alert.check_and_alert('TO_CONTROLLER', 'PASSIVE_COOLING', 40)[1] == 'Controller')
 
+    def test_check_and_alert_hi_active_cooling(self):
+        self.assertTrue(typewise_alert.check_and_alert('TO_CONSOLE', 'HI_ACTIVE_COOLING', -10)[0] == 'TOO_LOW')
```

Comment:
These assert statements convey the functionality of converting certain values to an array of two values.
If you were to state the purpose of check_and_alert in terms of data-conversion, what would you say?